tools/generate_binaries.sh

```diff
@@ -40,6 +40,9 @@ arch=$(uname -m)
 # Exclude fake packages
 exclusions="buildessential glibc parallel wayland xorg_proto"
 
+# Exclude packages that binary packages break
+exclusions="$exclusions docbook_xsl docbook_xml docbook"
```

Comment:
I'm still opposed to doing this.  If you are worried about the docbook catalog, let's try to create and append correctly in the binaries.

Reply:
Append, per say, doesn't work. With this, it could work, but the files won't be tracked. Xml has a very confusing format. It's like C and JSON. It has to have an end.

Reply:
What happens if we try to build all the binaries in a certain order.  Would that make sense?

Reply:
Maybe. We could also try the postinstall.

Reply:
I changed this.